common/djangoapps/third_party_auth/migrations/0006_samlproviderconfig_is_valid.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.12 on 2022-03-21 13:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('third_party_auth', '0005_auto_20210723_1527'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='samlproviderconfig',
+            name='is_valid',
```

Comment:
could we consider a slightly more descriptive field name like `was_valid_at` or something?

Reply:
its nice if boolean's are `is_` names and datetimes/timestamps are `_at` names. this field specifies a date in the past, rather than the current valid/invalid boolean state of the integration so, thats why I felt something like `was_valid_at` was more descriptive. open to other ideas and its certainly not enough reason to stop progress on the overall work.